### ISSUE TITLE
Export/Import list of dim/measures/object

### ISSUE THREAD

**Issue by countnazgul** [qlik-oss/corectl#440]
Is it possible to export/import a predefined list of dim/measures/objects?

The idea is to have a list of IDs somewhere and extract their layout in json format (the output file will be used in another import operation). 

I'm guessing that powershell/python/whatever script can be written that loops through the IDs list and use corectl to print the json and store the output. But was just wandering if corectl can do this for me ;)

**Comment by glooms** [comment]
Short answer: no. But, you could use the `--quiet` flag.
If you are using bash the following should work:
```bash
MEASURES=$(corectl measure ls -q | grep $1)
for m in $MEASURES; do
  corectl measure layout $m > "$m.json"
done
```
or if you prefer a oneliner:
```bash
MEASURES=$(corectl measure ls -q | grep "foo"); for m in $MEASURES; do corectl measure layout $m > "$m.json";done
```

Let me know if that helps @countnazgul !

**Comment by countnazgul** [comment]
Yep it helps. I was planning to use something like this as a workaround :)

Need to free some time (somehow) and try and submit PR with this functionality ;)

**Comment by glooms** [comment]
Nice! Could I just ask what the use case is?

You can specify a glob pattern when you import:
```bash
corectl object set obj*.json
```
So you can already specify exactly which "entities" to import using glob patterns.

The only support I might see missing is that you cannot export multiple entities at once. This would simplify the scripting a bit (excluding the need the `for` loop above) but I'm not sure it would satisfy your use case still.

**Comment by countnazgul** [comment]
Yep the import seems ok(ish) for what i have in mind.

If you want we can jump on a quick call at some point and will explain you what i have in mind as a workflow :)

**Comment by glooms** [comment]
Ye just ping us on branch slack (#qlik-core) :)
